code/controllers/subsystem/ticker.dm

```diff
@@ -228,6 +228,8 @@ var/datum/subsystem/ticker/ticker
 			if(M.stat != DEAD)
 				var/turf/T = get_turf(M)
 				if(T && T.z==1)
+					if(corebunker && istype(get_area(M), /area/turret_protected/ai))
```

Comment:
Should probbably technically be an area var but i doubt we'll want more nukeproof bunkers.